brave/brave-core
# Encrypt state for Brave Ads

Resolves https://github.com/brave/brave-browser/issues/11884

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (unit, browser, security tests) on

## Test Plan:

- Confirm ad_conversions, catalog, client, confirmations and notifications files in `Default/ads_service` are encrypted
- Confirm ad_conversions.json, catalog.json, client.json, confirmations.json and notifications.json files in `Default/ads_service` are removed for upgrade paths
- Confirm ads are delivered

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.
